lib/Adapter/TolerantParser/TolerantUpdater.php

```diff
@@ -115,7 +115,12 @@ private function updateUseStatements(Edits $edits, SourceCode $prototype, Source
             foreach ($node->getChildNodes() as $childNode) {
                 if ($childNode instanceof NamespaceUseDeclaration) {
                     foreach ($childNode->useClauses->getElements() as $useClause) {
-                        if ($useClause->namespaceName->getText() == $usePrototype->__toString()) {
+                        $cmp = strcmp($useClause->namespaceName->getText(), $usePrototype->__toString());
```

Comment:
Would be nice to somehow extract this to a query method, e.g. `->shouldBeInsertedBefore($useClause, $usePrototype)`, but then we should probably think about extracting this into a class. I think, short of opening up a can of refactoring, a comment would be sufficient as it's not immediately clear what's going on